doc/source/tune-config.rst

```diff
@@ -0,0 +1,78 @@
+Experiment Configuration
+========================
+
+
+Experiment Setup
+----------------
+
+There are two ways to setup an experiment - one via Python and one via JSON.
+
+The first is to create an Experiment object. You can then pass in either
+a single experiment or a list of experiments to `run_experiments`, as follows:
+
+.. code-block:: python
+
+    # Single experiment
+    run_experiments(Experiment(...))
+
+    # Multiple experiments
+    run_experiments([Experiment(...), Experiment(...), ...])
```

Comment:
Oh nice, did not know this was possible.

Reply:
after this PR :)